tests/test_main.py

```diff
@@ -178,5 +178,4 @@ def test_arthur_and_lancelot_have_potion(base_knights_config):
         "Arthur": 60,
         "Mordred": 10,
         "Red Knight": 0,
-    }
-
+    }
```

Comment:
Make sure that there are no changes in the `tests/test_main.py` file or other test files in the future that are out of the task scope